src/backend/libpq/rangerrest.c

```diff
@@ -294,7 +294,7 @@ static json_object *create_ranger_request_json(List *request_list, List *result_
 				break;
 			}
 			default:
-				elog(ERROR, "unrecognized objkind: %d", (int) kind);
+				elog(ERROR, "unrecognized object kind : %d", (int) kind);
```

Comment:
since this is a message to users, we should map this "kind" to string, so that users can know which kind is not supported.